flasc/data_processing/filtering.py

```diff
@@ -734,15 +790,21 @@ def get_power_curve(self, calculate_missing=True):
         return self.pw_curve_df
 
     def plot_farm_mean_power_curve(self, fm=None):
-        """Plot all turbines' power curves in a single figure. Also estimate
+        """Plot all turbines' power curves in a single figure.
+
+        Also estimate
         and plot a mean turbine power curve.
 
         Args:
             fm (FlorisModel): The FlorisModel object for the farm. If
               specified by the user, then the farm-average turbine power curve
               from FLORIS will be plotted on top of the SCADA-based power curves.
-        """
 
+        Returns:
+            fig, ax: The figure and axis objects of the
```

Comment:
Shouldn't this be a list?
```
Returns:
    fig: ...
    ax: ...
```
for consistency?

Reply:
I'm not sure which is correct, I'm going to look it up and go back through the cases where we return several variables

Reply:
https://google.github.io/styleguide/pyguide.html#doc-function-returns